CONTRIBUTING.md

```diff
@@ -75,36 +75,24 @@ changes.
 
 ### Developing and Running SQLFluff Locally
 
-To use your local development branch of SQLFluff, I recommend you use a virtual
-environment. e.g:
-
+The simplest way to set up a development environment is to use `tox`.
+First ensure that you have tox installed:
 ```shell
-python3 -m venv .venv
-source .venv/bin/activate
+python3 -m pip install -U tox
 ```
 
-> `python3 -m venv .venv` creates a new virtual environment (in current working
-> directory) called `.venv`.
-> `source .venv/bin/activate` activates the virtual environment so that packages
-> can be installed/uninstalled into it. [More info on venv](https://docs.python.org/3/library/venv.html).
-
-Once you are in a virtual environment, run:
-
+A virtual environment can then be created and activated by running:
 ```shell
-pip install -U tox
-pip install -Ur requirements.txt -Ur requirements_dev.txt
-pip install -e .
+tox -e py --devenv .venv
+source .venv/bin/activate
 ```
+(The `py` environment defaults to the python version used
+to install tox, however any version you want can be installed
+by replacing `py` with `py37`, `py39`, `dbt020-py38`, etc.)
```

Comment:
```suggestion
by replacing `py` with `py37`, `py39`, `dbt020-py38`, etc. If
you are planning development on or using the dbt templater
you may wish to chose one of the dbt environments.)

Windows users should call `.venv\Scripts\activate` rather than `source .venv/bin/activate`.
```